lib/base.rb

```diff
@@ -0,0 +1,30 @@
+class Base
```

Comment:
The Base class should contain common methods that a song, genre and artist share. This inheritance shortens code and also makes code more OOP. A mixin is good but inheritance is best suited here